frontend/packages/dev-console/integration-tests/features/getting-started/getting-started -to-dev-perspective.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Login to developer perspective
```

Comment:
Do we still require this scenario? 
Also there is an extra space in the file name.